addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -317,8 +324,9 @@ export class Many2XAutocomplete extends Component {
         if (this.lastProm) {
             this.lastProm.abort(false);
         }
-        this.lastProm = this.search(request);
-        const records = await this.lastProm;
+        const searchPromise = this.abortableSearch(request);
+        this.lastProm = searchPromise;
+        const records = await searchPromise.promise;
```

Comment:
Why don't we do that:

```suggestion
        this.lastProm = this.abortableSearch(request);
        const records = await this.lastProm.promise;
```